internal/response-consumer/handler.go

```diff
@@ -338,7 +340,7 @@ func checkSatStatusPartial(events *[]message.PlaybookSatRunResponseMessageYamlEv
 }
 
 type parsedMessageInfo struct {
-	Account         string
```

Comment:
the validator is no longer setting this field i.e. we can remove this altogether